spyder/plugins/variableexplorer/widgets/objectexplorer/objectexplorer.py

```diff
@@ -15,26 +15,23 @@
 
 # Third-party imports
 from qtpy.QtCore import Slot, QModelIndex, QPoint, QSize, Qt
-from qtpy.QtGui import QKeySequence, QTextOption
+from qtpy.QtGui import QTextOption
 from qtpy.QtWidgets import (
-    QAbstractItemView, QAction, QButtonGroup, QGroupBox, QHBoxLayout,
-    QHeaderView, QMenu, QMessageBox, QPushButton, QRadioButton, QSplitter,
-    QToolButton, QVBoxLayout, QWidget)
+    QAbstractItemView, QButtonGroup, QGroupBox, QHBoxLayout, QHeaderView,
+    QMessageBox, QPushButton, QRadioButton, QSplitter, QStyle, QToolButton,
+    QVBoxLayout, QWidget)
 
 # Local imports
 from spyder.api.config.fonts import SpyderFontsMixin, SpyderFontType
-from spyder.api.config.mixins import SpyderConfigurationAccessor
-from spyder.api.widgets.menus import SpyderMenu
+from spyder.api.widgets.mixins import SpyderWidgetMixin
 from spyder.config.base import _
-from spyder.config.manager import CONF
 from spyder.plugins.variableexplorer.widgets.basedialog import BaseDialog
 from spyder.plugins.variableexplorer.widgets.objectexplorer import (
     DEFAULT_ATTR_COLS, DEFAULT_ATTR_DETAILS, ToggleColumnTreeView,
     TreeItem, TreeModel, TreeProxyModel)
 from spyder.utils.icon_manager import ima
-from spyder.utils.qthelpers import (
-    add_actions, create_toolbutton, qapplication, safe_disconnect)
-from spyder.utils.stylesheet import PANES_TOOLBAR_STYLESHEET
+from spyder.utils.qthelpers import add_actions, qapplication
```

Comment:
```suggestion
from spyder.utils.qthelpers import qapplication
```

As before, `add_actions` shouldn't be necessary, so please replace its usage with the appropriate API method.